go_metrics.go

```diff
@@ -93,8 +93,6 @@ func writeRuntimeMetrics(w io.Writer) {
 
 func writeRuntimeMetric(w io.Writer, name string, sample *runtimemetrics.Sample) {
 	switch sample.Value.Kind() {
-	case runtimemetrics.KindBad:
-		panic(fmt.Errorf("BUG: unexpected runtimemetrics.KindBad for sample.Name=%q", sample.Name))
```

Comment:
This isn't good solution, since it may hide programming errors in the future. For example, if incorrect metric name with some typo is added to the `runtimeMetrics`, then this metric name will remain unnoticed.

The commit 447d235cbb84d47adde5cd96943eb0d0d44a8836 implements better solution - it logs unsupported Go runtime metrics from the `runtimeMetrics` at startup, so the user could upgrade Go builder in order to support these metrics.